llvm/test/Transforms/InstCombine/cheri-memcpy-64bit-cap.ll

```diff
@@ -0,0 +1,84 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
```

Comment:
1. Please commit the test case first with the existing output and then update it as part of the bug fix so it's clear what changes
2. Most of the attribute in here should be trimmable

Reply:
OK.  I'll do the changes with a force-push to get the commits in nice order.

Reply:
Should be fixed now?